x-pack/plugins/ml/public/application/overview/components/sidebar.tsx

```diff
@@ -59,7 +59,7 @@ export const OverviewSideBar: FC<Props> = ({ createAnomalyDetectionJobDisabled }
         <p>
           <FormattedMessage
             id="xpack.ml.overview.gettingStartedSectionText"
-            defaultMessage="Welcome to Machine Learning. Get started by reviewing our {docs} or {createJob}. We recommend using {transforms} to create feature indices for analytics jobs."
```

Comment:
I think it might be good to also delete `createJob: getCreateJobLink(createAnomalyDetectionJobDisabled),` line as well since we won't be using it.

Reply:
Thanks @qn895! I removed the line via ba380ce.